### ISSUE TITLE
Shadows don't work on Vulkan

### ISSUE THREAD

**Issue by moosichu** [huwb/crest-oceanrender#283]
**Describe the bug**
Shadows don't work on platforms which support Vulkan. This applies on Android and Linux.

**Log**
```
Shader error in 'UpdateShadow.compute': Unknown parameter type (54) for_LightShadowData at kernel UpdateShadow
``

**Versions**
Built-in pipeline on Unity 2018.4

**To Reproduce**
Enable shadows and Vulkan.

**Platform**
 - Editor and Standalone
 - Android/Windows/Linux

**Comment by moosichu** [comment]
Fixes by #282

**Comment by huwb** [comment]
Good work
